wandb/apis/public.py

```diff
@@ -1658,8 +1658,8 @@ def convert_objects(self):
 
         return objs
 
-    def __repr__(self):
-        return f"<Runs {self.entity}/{self.project}>"
+    def __repr__(self) -> str:
+        return f"<Runs {self.entity}/{self.project} ({len(self)})>"
```

Comment:
@gtarpenning could you make sure that using len on the runs object is safe?
we do this lazy eval so i want to make sure it works properly

Reply:
<img width="276" alt="Screen Shot 2023-05-04 at 1 59 02 PM" src="https://user-images.githubusercontent.com/19414170/236327789-4ddaa714-abd3-495d-9ae4-9eff99a1bfaf.png">

Like this? @kptkin